projects/discussion-ui/src/lib/components/discuss-all/discuss-all.component.ts

```diff
@@ -0,0 +1,158 @@
+import { CONTEXT_PROPS } from './../../services/discussion.service';
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { DiscussionService } from '../../services/discussion.service';
+import { TelemetryUtilsService } from './../../telemetry-utils.service';
+import * as CONSTANTS from './../../common/constants.json';
+/* tslint:disable */
+import * as _ from 'lodash'
+import { NSDiscussData } from '../../models/discuss.model';
+import { DiscussStartComponent } from '../discuss-start/discuss-start.component';
+import { BsModalService, BsModalRef } from "ngx-bootstrap/modal";
+
+/* tslint:enable */
+
+@Component({
+  selector: 'lib-discuss-all',
+  templateUrl: './discuss-all.component.html',
+  styleUrls: ['./discuss-all.component.scss']
+})
+export class DiscussAllComponent implements OnInit {
+
+  discussionList = [];
+  routeParams: any;
+  showStartDiscussionModal = false;
+  categoryId: string;
+  isTopicCreator = false;
+  showLoader = false;
+  currentFilter = 'recent'
+  currentActivePage: any;
+  fetchNewData: false;
+  bsModalRef: BsModalRef;
+
+  constructor(
+    public router: Router,
+    private route: ActivatedRoute,
+    private discussionService: DiscussionService, private modalService: BsModalService,
+    private telemetryUtils: TelemetryUtilsService) { }
+
+  ngOnInit() {
+    this.telemetryUtils.logImpression(NSDiscussData.IPageName.HOME);
+
+    this.route.params.subscribe(x => {
+      console.log('-------------xxxxxxxxxx------------')
```

Comment:
Remove the console commands if not required now.

Reply:
This one is old code. When i will raise PR for all discussion page this code will be removed.